doc/nrf/libraries/caf/sensor_data_aggregator.rst

```diff
@@ -0,0 +1,79 @@
+.. _caf_sensor_data_aggregator:
+
+CAF: Sensor data aggregator module
+##################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The |sensor_data_aggregator| of the :ref:`lib_caf` (CAF) is a simple module responsible for aggregating sensor data in form of ``sensor_event`` and passing them further in packages.
+It can be used in multi-core chips to reduce power consumption.
```

Comment:
```suggestion
It can be used in both single-core and multi-core SoCs to reduce power consumption.
```

Reply:
changed a bit because in single-core power consumption level is not reduced.